.appveyor.yml

```diff
@@ -20,7 +16,13 @@ build: false
 
 # Install scripts
 install:
-  - pip install --upgrade pip
+  - set PATH=%CONDA_INSTALL_LOCATION%;%CONDA_INSTALL_LOCATION%\scripts;%PATH%;
+  - conda config --set always_yes true  # else AppVeyor will hang forever waiting for user ([y]/n)
+  - conda update --all
```

Comment:
I think it might be enough to do `conda update conda` and perhaps save a bit of time

Reply:
Ah, yes, since we pip install the rest. Gotcha.

Reply:
Nope since we create an env below. The root env is never used